packages/jest-reporters/src/CoverageReporter.ts

```diff
@@ -455,7 +455,7 @@ export default class CoverageReporter extends BaseReporter {
 
           const converter = v8toIstanbul(
             res.url,
-            fileTransform?.wrapperLength ?? 0,
```

Comment:
this should still pass `0`, as it defaults (on older node 10, which we support) to Node's wrapper length (which is wrong for Jest as we inject `jest` for instance): https://github.com/istanbuljs/v8-to-istanbul/blob/aac4e36e64c695c13abca9027b62b68b3e47db72/lib/v8-to-istanbul.js#L28